InvenTree/InvenTree/admin.py

```diff
@@ -31,3 +31,15 @@ def export_resource(self, obj):
                 row[idx] = val
 
         return row
+
+    def get_fields(self, **kwargs):
+        """Return fields, with some common exclusions"""
+
+        fields = super().get_fields(**kwargs)
+
+        fields_to_exclude = [
+            'metadata',
```

Comment:
But doesn't that mean one could not import metadata anymore if one wants?

Reply:
Yes, for now at least. The existing implementation was inconsistent - metadata was excluded from import on a bunch of models already.